sevntu-checks/pmd-test.xml

```diff
@@ -60,16 +60,16 @@
                    test method.
                  CheckstyleRegressionTest is not a test. -->
             <property name="violationSuppressXPath"
-                      value="//ClassOrInterfaceDeclaration[@Image='AllChecksTest'
-              or @Image='CheckstyleRegressionTest']"/>
+                      value="//ClassOrInterfaceDeclaration[@SimpleName='AllChecksTest'
```

Comment:
All `Image` attributes should be replaced with `SimpleName` after PMD 6.22:
https://javadoc.io/static/net.sourceforge.pmd/pmd-java/6.22.0/net/sourceforge/pmd/lang/java/ast/ASTAnyTypeDeclaration.html#getImage()